<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QWidget" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Time</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="Panel1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="Label1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Input Approx. Log Start Time (GPST)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDateEdit" name="TimeY1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="TimeH1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="BtnFileTime">
        <property name="text">
         <string>File Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnOk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
